service/src/test/resources/ErlookupOptionsWithMultipleFoundInput.json

```diff
@@ -0,0 +1,39 @@
+{
+    "msgParams": {
+        "meta": {
+            "type": "EiffelCompositionDefinedEvent",
+            "version": "3.0.0",
+            "tags": [
+                ""
+            ],
+            "source": {
+                "domainId": "",
+                "host": "",
+                "name": "",
+                "uri": ""
+            }
+        }
+    },
+    "eventParams": {
+        "data": {
+            "name": "required",
+            "version": "",
+            "customData": [
+
+            ]
+        },
+        "links": [{
+            "type": "ELEMENT",
+            "%lookup%": {
+                "eventType": "EiffelArtifactCreatedEvent",
+                "properties": [{
+                    "data.identity": "pkg:maven/swdi.up/CXP102051_22@R21EK?type=xml&classifier=test"
+                }],
+                "options": {
+					"failIfNoneFound": "true",
+					"failIfMultipleFound": "true"
+				}
```

Comment:
For all the examples mentioned here, I would like to see multiple lookups and the criterias are working in all possible scenarios

Reply:
Added multiple lookups in all examples with possible scenarios. If I missed any scenario please let me know.